src/utils/changeScreenState.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-param-reassign */
+import style from '../App.module.css';
+
+export default function changeScreenState(props) {
+  if (props.e.target.innerText === 'fullscreen_exit') {
+    document.body.style.overflow = 'auto';
+    // eslint-disable-next-line no-param-reassign
```

Comment:
Fix here and below

Reply:
Done -> https://github.com/SorokinTim/covid-dashboard-rss/pull/20/commits/61db95e8483c0066d7fd6f5943d9d16b49732590